packages/client/src/components/TransactionList.js

```diff
@@ -1,78 +1,47 @@
 import React from "react";
-import { useModalContext } from "../contexts";
-import { getFlowscanUrlForTransaction, shortenAddr } from "../utils";
+import { useModalContext } from "contexts";
 import TransactionStatusIcon from "./TransactionStatusIcon";
 
-const FALLBACK_RECIPIENT_NAME = "No name";
+const maxWidths = {
```

Comment:
We should avoid rendering with hardcode pixel values unless absolutely necessary. Anyway we can solve the problem with flex-box or bulma columns?

Reply:
its necessary to define the widths here so that `intent` (Info column) can be take up all available width as the strings can be quite long.

Reply:
Would using flex-grow help?